plugins/operator-avg/src/test/scala/com/stratio/sparkta/plugin/operator/avg/AvgOperatorSpec.scala

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (C) 2015 Stratio (http://stratio.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.stratio.sparkta.plugin.operator.avg
+
+import org.scalatest.{Matchers, WordSpec}
+
+class AvgOperatorSpec extends WordSpec with Matchers {
```

Comment:
@danielcsant Why? It is extending from WordSpec and @RunWith is not needed.
Furthermore JUnitRunner is a Java's library.